.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+# Basic dependabot.yml file with
+# minimum configuration for two package managers
+
+version: 1
```

Comment:
Shouldn't it be version: 2? We have that in starter and its defined like that in all examples